modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -175,6 +175,16 @@ const get = (dom: DOMUtils) => {
       }
     },
 
+    lang: {
+      inline: 'span',
+      clear_child_styles: true,
+      remove_similar: true,
+      attributes: {
+        'lang': '%value',
+        'data-mce-lang': (vars) => vars?.customValue ?? null
```

Comment:
Right, this has impact on RTC we only support a smaller set of formatter api things and functions on these values is not one of them. I'm adding as separate ticket to expand the rtc formatter to include this as well. Curious why is this data attribute needed?

Reply:
The data attribute is there for languages like "English (US Medical)" that are useful variants to be able to tell the difference between in your HTML, but don't have valid ISO language codes that could be put into a `lang` attribute.

For what it's worth, the main reason we want to note things like "this content uses medical terms" in the HTML is so that spellchecker pro can know to treat them differently.